.github/workflows/ci.yml

```
@@ -0,0 +1,43 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  format:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout your repository using git
+        uses: actions/checkout@v4
+      - name: Setup Bun
+        uses: oven-sh/setup-bun@v1
+      - name: Install dependencies
+        run: bun install
+      - name: Format
+        run: bun format
```

```suggestion
        run: bun format:check
```

`format`スクリプトは`rettier --ignore-path .gitignore --write . --cache`であり、フォーマットの修正用スクリプトなので、フォーマットのチェックはしません。よって、新しく以下のような`format:check`スクリプトを作成してそれを使うべきだと思います。

`package.json`:
```json
{
  "scripts": {
    "format:check": "prettier --ignore-path .gitignore --check . --cache"
  }
}
```